<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>505</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>450</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QLabel" name="ballGame_image">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="quitBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>150</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logInBtn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
     <widget class="QLabel" name="adminLogInLabel">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>350</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Administrator Log In</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shoppingCartBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>340</y>
        <width>150</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Shopping Cart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewStadiumBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>150</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View Stadiums</string>
      </property>
     </widget>
     <widget class="QPushButton" name="planVacationBtn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>150</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Plan A Vacation</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modifySouvenirPage">
     <widget class="QPushButton" name="modifySouvenirBtn">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>220</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify Souvenir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addSouvenirBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add Souvenir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteSourvenirBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Delete Souvenir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adminPage">
     <widget class="QPushButton" name="modifySouvenirListBtn">
      <property name="geometry">
       <rect>
        <x>219</x>
        <y>230</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify Souvenir List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifyStadiumListBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Modify Stadium List</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modifyStadiumPage">
     <widget class="QPushButton" name="modifyStadiumBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>240</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify Stadium</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStadiumBtn">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>190</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add Stadium</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="planVacationPage">
     <widget class="QPushButton" name="shortestAngelBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Shortest Trip from Angel Stadium</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customizeShortestBtn">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>230</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Customize Shortest Trip</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MSTBtn">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>280</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimum Spanning Tree</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
